gen3-client/g3cmd/upload.go

```diff
@@ -134,9 +152,10 @@ func init() {
 			if len(multipartFilePaths) > 0 {
 				log.Println("Multipart uploading....")
 				for _, filePath := range multipartFilePaths {
-					fileInfo, err := ProcessFilename(uploadPath, filePath, includeSubDirName)
+					// NOTE @mpingram -- metadata upload will not work for multipart uploads until Shepherd API supports multipart uploads
+					fileInfo, err := ProcessFilename(uploadPath, filePath, includeSubDirName, false)
```

Comment:
i'd prefer to also check if we are using shepherd here and print some msgs && exiting if that is the case